.github/renovate.json

```diff
@@ -56,12 +56,12 @@
 			"matchUpdateTypes": ["digest", "patch", "minor", "major"]
 		},
 		{
-			"matchPackagePatterns": [
-				"@insurgent/conventional-changelog-preset",
-				"@insurgent/commitlint-config"
-			],
 			"groupName": "semantic-release related packages",
-			"matchUpdateTypes": ["digest", "patch", "minor", "major"]
+			"matchUpdateTypes": ["digest", "patch", "minor", "major"],
+			"matchPackageNames": [
+				"/@insurgent/conventional-changelog-preset/",
```

Comment:
The middle `/` might cause an issue, need to check the docs/code to see if that would be the case before merging.